tests/performance-tests/atala-performance-tests-k6/README.md

```diff
@@ -1,20 +1,23 @@
-## Prerequisites
+# Prerequisites
 
 - [k6](https://k6.io/docs/getting-started/installation)
 - [NodeJS](https://nodejs.org/en/download/)
 - [Yarn](https://yarnpkg.com/getting-started/install) (optional)
 
-## Installation
+# Installation
 
 **Install dependencies**
 
 Clone the generated repository on your local machine, move to the project root folder and install the dependencies defined in [`package.json`](./package.json)
 
+*NOTE*: The Project has a dependency on `input-output-hk/prism-typescript-client` which is a private repository.
```

Comment:
:::note

This project has a dependency on `input-output-hk/prism-typescript-client` which is a private repository.